helpdesk/templates/teams/team_list.html

```diff
@@ -0,0 +1,20 @@
+{% extends "layouts/base_app.html" %}
+{% load render_table from django_tables2 %}
+{% load crispy_forms_tags %}
+
+{% block page_title %}Team Attendance{% endblock %}
+{% block title %}Team Attendance{% endblock %}
+
+{% block page_buttons %}
+{#    <input class="input" type="date">#}
```

Comment:
Question: Is this meant to be commented out?

Reply:
This is a WIP thing. I wanted to add the option to filter by day. Not sure if I should keep it. We'd also want to make sure that teams which have dropped out ignore the filter.